src/components/Header/Header.tsx

```diff
@@ -0,0 +1,76 @@
+'use client'
+
+import { BottomNavigation, BottomNavigationAction, Box } from '@mui/material'
+import React from 'react'
+
+interface Props {}
+
+const style = {
+  link: {
+    fontSize: 14,
+    fontWeight: '600',
+    cursor: 'pointer',
+    borderRadius: '6px',
+    paddingX: '0.8rem',
+    paddingY: '0.4rem',
+    '&:hover': {
+      transform: 'translateY(0.4rem)',
+      backgroundColor: 'black',
+      color: 'white',
+    },
+    '&:active': {
+      transform: 'scale(0.90) translateY(0.4rem)',
+    },
+    '& .Mui-selected': {
+      color: '#2563eb',
+    },
+    transition: 'all 240ms cubic-bezier(0.4, 0.4, 0.4, 0.4)',
+  },
+}
```

Comment:
Not that I think this is necessarily wrong, but more of a question for the whole group, do we want to separate styling code into it's own file, even if it's just a `.ts` file that exports this object?

Reply:
I agree, we should have a separte file for this css.

Reply:
I have separated it to a styles folder. Check it out.